Profiles/cio-Symptom.xml

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It is really curious that no date/time concept from the data set is present (yet) in the transactions, and hence in this profile. It seems that a Symptom without time stamp would give a lot less information, since you wouldn't know when it occurred, or since when it occurs. According to the ticket, this is still an ongoing discussion?

Reply:
Yes, this is an ongoing discussion. I would expect the StatusDate to be included in the transaction to capture this information. Specifically because RegistratieDatumTijd is not fit to capture this information.